configure.ac

```diff
@@ -828,6 +828,36 @@ AC_CHECK_FUNCS_ONCE([rdma_lib_reset])
 AC_CHECK_LIB([ibverbs], [ibv_cmd_poll_cq], [VERBS_FLAGS="$VERBS_FLAGS -libverbs"],
 		[])
 AC_SUBST([VERBS_FLAGS])
+#
+#
+#
+AC_ARG_ENABLE( [mpqp],
+    AC_HELP_STRING(
+        [--enable-mpqp],
+        [Enable Multi Packet RQ QP experimental feature (default=no)]))
+#
+AC_CHECK_DECLS([IBV_EXP_DEVICE_ATTR_VLAN_OFFLOADS,
+		IBV_EXP_DEVICE_ATTR_MAX_CTX_RES_DOMAIN,
+		IBV_EXP_CQ_RX_UDP_PACKET,
+		MLX5_CQE_L3_HDR_TYPE_MASK,
+		MLX5_CQE_L4_OK,
+		MLX5_CQE_L4_HDR_TYPE_UDP],
+		[can_mp_rq=yes],
```

Comment:
This is not good
Even if one of the symbols exists it will change can_mp_rq to true